test/libsolidity/syntaxTests/conversion/implicit_conversion_of_super_in_operators.sol

```diff
@@ -30,25 +30,25 @@ contract C {
-// TypeError 2271: (117-129): Operator | not compatible with types type(contract super C) and contract C.
-// TypeError 2271: (139-151): Operator & not compatible with types type(contract super C) and contract C.
-// TypeError 2271: (162-174): Operator * not compatible with types type(contract super C) and contract C.
-// TypeError 2271: (184-196): Operator / not compatible with types type(contract super C) and contract C.
-// TypeError 2271: (206-218): Operator % not compatible with types type(contract super C) and contract C.
-// TypeError 2271: (228-240): Operator - not compatible with types type(contract super C) and contract C.
-// TypeError 2271: (250-262): Operator + not compatible with types type(contract super C) and contract C.
-// TypeError 2271: (272-285): Operator ** not compatible with types type(contract super C) and contract C.
-// TypeError 2271: (296-309): Operator == not compatible with types type(contract super C) and contract C.
-// TypeError 2271: (319-332): Operator != not compatible with types type(contract super C) and contract C.
-// TypeError 2271: (342-355): Operator >= not compatible with types type(contract super C) and contract C.
-// TypeError 2271: (365-378): Operator <= not compatible with types type(contract super C) and contract C.
-// TypeError 2271: (388-400): Operator < not compatible with types type(contract super C) and contract C.
-// TypeError 2271: (410-422): Operator > not compatible with types type(contract super C) and contract C.
-// TypeError 2271: (433-446): Operator || not compatible with types type(contract super C) and contract C.
-// TypeError 2271: (456-469): Operator && not compatible with types type(contract super C) and contract C.
+// TypeError 2271: (49-62): Built-in binary operator << cannot be applied to types type(contract super C) and contract C.
+// TypeError 2271: (72-85): Built-in binary operator >> cannot be applied to types type(contract super C) and contract C.
+// TypeError 2271: (95-107): Built-in binary operator ^ cannot be applied to types type(contract super C) and contract C.
+// TypeError 2271: (117-129): Built-in binary operator | cannot be applied to types type(contract super C) and contract C.
+// TypeError 2271: (139-151): Built-in binary operator & cannot be applied to types type(contract super C) and contract C.
+// TypeError 2271: (162-174): Built-in binary operator * cannot be applied to types type(contract super C) and contract C.
+// TypeError 2271: (184-196): Built-in binary operator / cannot be applied to types type(contract super C) and contract C.
+// TypeError 2271: (206-218): Built-in binary operator % cannot be applied to types type(contract super C) and contract C.
+// TypeError 2271: (228-240): Built-in binary operator - cannot be applied to types type(contract super C) and contract C.
+// TypeError 2271: (250-262): Built-in binary operator + cannot be applied to types type(contract super C) and contract C.
```

Comment:
We're changing general mentions of operators into built-in operators in anticipation of whatever we call operator overloading here, right? As in, we'll be able to bind a custom function to a bulit-in operator that will be executed only for custom and defined types?
Anyway, the reason I'm asking this is because I assume that once that feature is merged, we'll be updating some of the messages to reflect user defined operators vs built-in ones?

Reply:
> We're changing general mentions of operators into built-in operators in anticipation of whatever we call operator overloading here, right? As in, we'll be able to bind a custom function to a bulit-in operator that will be executed only for custom and defined types?

Yes.

> Anyway, the reason I'm asking this is because I assume that once that feature is merged, we'll be updating some of the messages to reflect user defined operators vs built-in ones?

We're actually mostly adding new messages. The existing ones still work in most cases and where they don't the clarifications make sense also outside of the PR. Which is why I'm submitting these corrections separately. They add a lot of noise in the operators PR.